### PR TITLE
feat(ExclusiveForm): add form for exclusive features

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add exclusive features form to request a demo.
- TODO: final content, test.

image

### CODE REVIEW

**Comment by cbernat** [comment]
👍 I take your word in tests and content.

File: src/i18n/es.js [hunk 1]
```diff
@@ -113,8 +113,23 @@ export default {
   },
   empty_notification_text: `No tienes notificaciones pendientes.`,
   exclusive_form: {
+    advisory: 'Asesoría y Consultoría.',
     breadcrumb: 'Planes',
     breadcrumb_url: `${urlBuyMonthly}`,
+    dedicated_ip: 'IP Dedicada',
+    demo: 'Solicitar una demo',
+    description: 'Características especiales pensados para los usuarios más exigentes. Desarrollo de funcionalidades a medida, atención personalizada y más..',
+    design: 'Diseño, maquetacion y envio de Emails',
```
Comment (cbernat) [inline src/i18n/es.js:122]:
```suggestion
    design: 'Diseño, maquetación y envío de Emails',
```

**Comment by cbernat** [comment]
Please add this form into special features to be easy to access.

File: src/i18n/es.js [hunk 2]
```diff
@@ -112,6 +112,26 @@ export default {
     title: 'Suma contactos a tus Listas con Formularios personalizados',
   },
   empty_notification_text: `No tienes notificaciones pendientes.`,
+  exclusive_form: {
```
Comment (cbernat) [inline src/i18n/es.js:115]: @mrunge86 Here is the content for the form.

**Comment by fcoronelmakingsense** [comment]
update: app content and changes

image

**Comment by fcoronelmakingsense** [comment]
Add in new features

image

File: src/components/DopplerPlus/ExclusiveForm.js [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+  PhoneFieldItem,
+  SubmitButton,
+} from '../form-helpers/form-helpers';
+import { useIntl } from 'react-intl';
+
+const ExclusiveForm = ({ dependencies: { dopplerLegacyClient, appSessionRef } }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  // TODO: apply content
```
Comment (lucianaDilizio) [inline src/components/DopplerPlus/ExclusiveForm.js:21]: You should delete this comment.

**Comment by lucianaDilizio** [comment]
How about add tests for this in a new commit? You can do it based in tests for agencies form.

File: src/components/NewFeatures/NewFeatures.js [hunk 4]
```diff
@@ -67,6 +67,12 @@ const NewFeatures = () => {
             <Link to={'/billing/invoices'}>Ir a Facturas Emitidas</Link>
           </div>
         </div>
+        <div className="dp-block-wlp dp-box-shadow m-t-36">
+          <div className="m-l-24">
+            <h3>Formulario de caracteristicas exclusivas</h3>
```
Comment (cbernat) [inline src/components/NewFeatures/NewFeatures.js:72]:
```suggestion
            <h3>Formulario de características exclusivas</h3>
```

File: src/i18n/es.js [hunk 5]
```diff
@@ -131,6 +131,29 @@ export default {
...
+    advisory: 'Asesoría y Consultoría.',
+    breadcrumb: 'Planes',
+    breadcrumb_url: `${urlBuyMonthly}`,
+    custom_onboarding: 'Onboarding personalizado',
+    custom_reports: 'Reportes a medida',
+    dedicated_ip: 'IPs Dedicadas',
+    demo: 'ENVIAR',
+    description: '¿Necesitas funcionalidades especiales para tu negocio? Cuéntanos cuáles son y podremos ayudarte con un Plan pensado para ti.',
+    design_layout_emails: 'Diseño, maquetacion y envio de Emails',
```
Comment (cbernat) [inline src/i18n/es.js:143]:
```suggestion
    design_layout_emails: 'Diseño, maquetación y envío de Emails',
```

**Comment by cbernat** [comment]
You could add some tests as @lucianaDilizio suggested.

**Comment by cbernat** [comment]
Nice tests 😍

File: src/components/DopplerPlus/ExclusiveForm.test.js [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+      fireEvent.submit(submitButton);
+    });
+
+    // Assert
+    await waitFor(() => {
+      expect(getByText('validation_messages.error_required_field')).toBeInTheDocument();
+      expect(container.querySelector('.dp-wrap-confirmation')).toBeNull();
+    });
+  });
+});
```
Comment (lucianaDilizio) [inline src/components/DopplerPlus/ExclusiveForm.test.js:231]: 💪

**Pull request merged**